src/templates/base.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<title>{% block title %}{% endblock %} - Festify</title>
```

Comment:
This field is used for the page title but you need to fill in the child blocks as well. At the moment is empty so it just shows " - Festify" for all pages